packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -552,17 +681,22 @@ export interface DocsQAProps {
    *  }
    * ```
    */
-  chunkFormatter: (props: { doc: ScoredChunk }) => Node;
+  chunkFormatter: (props: { doc: ScoredChunk<ChunkMetadata> }) => Node;
```

Comment:
can you make `DefaultFormatter` the default?

Reply:
I think so?  @NickHeiner Is this a reasonable way to do this?

Reply:
Oh, just saw your comment.  Well it works, so let's go with it!